### PR TITLE
[example] Writing HTML code in PHP

### PR DESCRIPTION
**Pull request by szepeviktor**
See https://github.com/szepeviktor/Toolkit4WP/blob/master/src/helpers.php#L17
or https://github.com/decodelabs/tagged

```php
$results['html'] = tag(
    'h6',
    [ 'class' => 'toolbelt-divider' ],
    sprintf( esc_html__( 'Page %d', 'wp-toolbelt' ), $page )
);
```

### CODE REVIEW

File: modules/infinite-scroll/module.php [hunk 1]
```diff
@@ -152,7 +152,12 @@ function toolbelt_is_active() {
 	 * Does not work on archives or other post types.
 	 * Simplifies the code massively.
 	 */
-	return is_home();
+	if ( ! is_home() ) {
+		return false;
+	}
+
+	return true;
```
Comment (szepeviktor) [inline modules/infinite-scroll/module.php]: How could this get in this PR???

**Comment by szepeviktor** [comment]
esc_html and sprintf should switch place

**Comment by BinaryMoon** [comment]
Thanks for the input. I've been thinking about this quite a bit since I really appreciate all the improvements that have happened because of your input, but I don't think I want to work with this.

I like seeing the html - I think it's clearer, and it follows WordPress conventions. I appreciate it's not to everyones taste but I feel more comfortable working this way so I think I'll stick with it for now.

**Comment by szepeviktor** [comment]
> it follows WordPress conventions.

Yes, you are right.

I'd like my projects to go with OOP.

**Pull request closed without merging**